pages/common/sc-im.md

```diff
@@ -0,0 +1,22 @@
+# SC-IM
+
+> A curse based, vim-like spreadsheet calculator.
```

Comment:
Shouldn't this be `curses` or `ncurses`?

Reply:
Fixed: changed to *curses*